ts/features/newWallet/store/actions/cards.ts

```diff
@@ -0,0 +1,21 @@
+import { ActionType, createStandardAction } from "typesafe-actions";
+import { WalletCard } from "../../types";
+
+export const walletUpsertCard =
+  createStandardAction("WALLET_UPSERT_CARD")<WalletCard>();
+
+export const walletAddCards =
+  createStandardAction("WALLET_ADD_CARDS")<ReadonlyArray<WalletCard>>();
+
+export const walletRemoveCard =
+  createStandardAction("WALLET_REMOVE_CARD")<WalletCard["key"]>();
+
+export const walletRemoveCards = createStandardAction("WALLET_REMOVE_CARDS")<
+  ReadonlyArray<WalletCard["key"]>
+>();
```

Comment:
Could we have only one action to remove card like `walletAddCards` ?

Reply:
Removed in https://github.com/pagopa/io-app/pull/5604/commits/59ee875436f1fa1dd371149243d7435c11ebe6df